src/test/java/unit/controller/AuthControllerUnitTest.java

```diff
@@ -0,0 +1,70 @@
+package unit.controller;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.controller.AuthController;
+import org.kainos.ea.dao.AuthDao;
+import org.kainos.ea.exception.DatabaseConnectionException;
+import org.kainos.ea.exception.FailedToEncryptTokenException;
+import org.kainos.ea.exception.FailedToGenerateTokenException;
+import org.kainos.ea.exception.FailedToLoginException;
+import org.kainos.ea.model.Login;
+import org.kainos.ea.service.AuthService;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import javax.ws.rs.core.Response;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+public class AuthControllerUnitTest {
+    AuthService authService = Mockito.mock(AuthService.class);
+    AuthDao authDao = Mockito.mock(AuthDao.class);
+    AuthController authController = new AuthController(authService);
+
+    Login login = new Login(
+            "admin",
+            "admin"
+    );
+    @Test
+    void login_shouldReturnOK_whenLoginSuccessful() throws FailedToLoginException, FailedToGenerateTokenException, FailedToEncryptTokenException {
+        String token = "4677678787878";
+        Login validLogin = new Login(
+                "keelan@gmail.com",
+                "Scott"
+        );
+        Mockito.when(authService.login(validLogin)).thenReturn(token);
```

Comment:
```suggestion
        Mockito.when(authService.login(login)).thenReturn(token);
```

this is a unit test so you can mock the dao to return a token, you don't need actual db credentials